packages/sandbox/src/sandbox.ts

```diff
@@ -36,6 +36,7 @@ export type SandboxOptions = {
   identifier?: string;
   format?: ClientConfigFormat;
   profile?: string;
+  disableWatcher?: boolean;
```

Comment:
In general, I prefer "positive booleans" rather than "negative booleans" because it avoids double negatives like what you have above with `!options.disableWatcher`. Instead, this could be `watchForChanges` with a default of true and then `options.watchForChanges` reads better.